### ISSUE TITLE
Deploy Specific Scripts

### ISSUE THREAD

**Issue by jasonbyrne** [stackpath/edgeengine-cli#10]
The Firebase CLI is really nice because it allows you the option to only deploy certain things. Let's say only one script change. There is no reason to deploy all.

firebase deploy --only functions

See this doc:
https://firebase.google.com/docs/cli/#partial_deploys

Would be great to have this same feature for Edge Engine.

**Comment by klaude** [comment]
I agree that execution could get pretty heavy after you get a bunch of scripts on your stacks. I'll bring this one up with the team and see how best to do this one.

**Comment by klaude** [comment]
Going off Firebase's example, what if we added a `--only` flag that takes a comma-separated list of script IDs? The ID would have to exist in the `edgeengine.json` file so the tool knows where to deploy the script too. This may be better after getting deploying scripts by name.

**Comment by jasonbyrne** [comment]
It would better than nothing, but of course as long as the script IDs are GUIDs then it's never going to be anything that can just be typed in. Every time you'd have to go into edgeengine.json and copy and paste it. So it's better than nothing but wouldn't be idea. Even if edgeengine.json had an "alias" or "tag" field that had no context outside of just locally that would be better. The CLI could just loop through what is in the config to filter the list of scripts.

**Comment by klaude** [comment]
I like that. Ideally, I'd like to support script slugs on the backend then this tool can uses those. Aliases in `edgeengine.json` will work in a pinch. I'll talk to our product team tomorrow to make sure they're cool with adding site script slugs. I can't see why they wouldn't be. After slugs are in place we can get cranking on this feature for ya.

**Comment by jasonbyrne** [comment]
Should this be closed since I did a PR for this and it was deployed?
